cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -17,21 +17,32 @@
         </span>
     </button>
     <div class="m-global-search_content" aria-expanded="false">
-        <form class="m-global-search_content-form">
+        <form class="m-global-search_content-form"
+              action="http://search.consumerfinance.gov/search"
+              method="get">
+            {# Required hidden input for CFPB search. #}
+            <input type="hidden"
+                           name="utf8"
+                           value="✓">
```

Comment:
http://stackoverflow.com/questions/27338154/why-do-some-websites-have-utf8-in-their-title